cmd/frontend/graphqlbackend/recorded_commands.go

```diff
@@ -31,6 +33,13 @@ type RecordedCommandsArgs struct {
 }
 
 func (r *RepositoryResolver) RecordedCommands(ctx context.Context, args *RecordedCommandsArgs) (graphqlutil.SliceConnectionResolver[RecordedCommandResolver], error) {
+
+	err := auth.CheckCurrentUserIsSiteAdmin(ctx, r.db)
+
+	if err != nil {
+		return nil, errors.New("only site admins can view recorded commands")
```

Comment:
I think it's fine to just return the `err` here to be consistent with the other resolvers:

```suggestion
		return nil, err
```